src/core/ext/filters/client_channel/OWNERS

```diff
@@ -1,4 +1,4 @@
 set noparent
 @markdroth
-@dgquintas
+@ncteisen
```

Comment:
Alex would probably be a better choice here.  Given his work on the c-ares resolver, he probably works on the client channel code more than Noah does.

Reply:
Done.